contracts/Vesting.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./DPT.sol";
```

Comment:
You are not using it, why you imported that?

Reply:
ReentrancyGuard is using in function "join"
Ownable - is using in function "startVesting" and "stopVesting"